components/form/UploadImageInput.tsx

```diff
@@ -8,6 +8,7 @@ type UploadImageProps = {
   setImageSrc: (src: string) => void;
   setError?: (error: string) => void;
   isRequired?: boolean;
+  imageType: 'item' | 'profile';
```

Comment:
Yep, this works and I think it's a pretty elegant solution. I'd say just add some information to the [JS Docs](https://github.com/enBloc-org/kindly/blob/dev/.github/BEST_PRACTICE.md#documenting-through-js-docs) for this component. At the moment it looks like it's actually missing details on some of the other props so if you could add those as well it would be good refactor of the documentation:

```typescript
/**
 * @description this function will add any image to our project storage.
 * @param isRequired is an optional parameter that accepts a boolean. It will consider the value of false unless explicitly assigned the value of true.
 */

const UploadImageInput: React.FC<UploadImageProps> = ({
  setImageSrc,
  setError,
  isRequired: isRequired = false,
  imageType,
}) => {
```